shared/README.md

```diff
@@ -0,0 +1,110 @@
+# Mobility Data Specification: **Shared Definitions**
+
+This specification contains a collection of definitions, data types, and vehicle state machine information used to specify the digital relationship between *mobility as a service* Providers and the Agencies that regulate them.  These definitions are used by the other MDS APIs, including the Provider API, the Agency API, and the Policy API.
+
+* Date 11 May 2020
+* Version: ALPHA
+
+## Table of Contents
+
+* [Timestamps](#timestamps)
+* [Strings](#strings)
+* [UUIDs](#uuids)
+* [Vehicle Types](#vehicle-types)
+* [Propulsion Types](#propulsion-types)
+* [Vehicle States](#vehicle-states)
+* [Vehicle Events](#vehicle-events)
+* [Costs and Currencies](#costs-and-currencies)
+
+## Timestamps
+
+A `timestamp` refers to integer milliseconds since Unix epoch.
+
+## Strings
+
+All String fields, such as `vehicle_id`, are limited to a maximum of 255 characters.
+
+## UUIDs
+
+Object identifiers are described via Universally Unique Identifiers [(UUIDs)](https://en.wikipedia.org/wiki/Universally_unique_identifier).  For example, the `device_id` field used to uniquely identify a vehicle is a UUID.
+
+MDS uses Version 1 UUIDs.
```

Comment:
Just out of curiosity, why are we using UUIDv1?  Any concern that is might have privacy implications for future uses due to MAC address linkage?

Reply:
I pushed for UUIDv4, but that would require replacing all prior in-use UUIDv1s, so it was shot down.